### ISSUE TITLE
Vive Tracker binding pose bindings are broken

### ISSUE THREAD

**Issue by wirelessdreamer** [ValveSoftware/openvr#979]
![image](https://user-images.githubusercontent.com/140221/50541946-d9d65d00-0b7e-11e9-9d84-7701f2ebbcb8.png)

When left and right hand raw are set, trackers should not have any rotation or position offsets applied to them, just like before the new input system screwed all tracker related things up. applying them in recent SteamVR builds no longer has any effect, and an offset is now applied to tracker rotation, so they no longer match up . . . again.

Currently using SteamVR Beta 1.2.3

**Comment by wirelessdreamer** [comment]
additionally, how do we define new poses in our openvr compatable games? i want a pose i can apply for the hyperblaster, and topshot elite, and sony sharpshooter, etc. the current setup is horribly confusing for users, we shouldn't make them jump through hoops to use basic controllers that aren't a vive controller

**Comment by mellott124** [comment]
Agreed.  This is a mess.  Struggling with the same issues you've mentioned.

**Comment by wirelessdreamer** [comment]
I'm currently using VIU's controller binding to handles poses, and don't have steamvr 2.0 support on my roadmap because of this. if i can rebind buttons but not poses it makes the functionality pretty pointless IMO.
